### ISSUE TITLE
Android build failed for Flutter 1.12.13+hotfix.5

### ISSUE THREAD

**Issue by Sleevezipper87** [brianegan/flutter_stetho#33]
Hi!

Android builds seem to fail when using the latest Flutter version (1.12.13+hotfix.5) and Stetho 0.5.0.

`flutter build apk --release`

```
(...)
Execution failed for task ':flutter_stetho:verifyReleaseResources'.
> java.util.concurrent.ExecutionException: com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed
  /Users/vagrant/git/build/flutter_stetho/intermediates/res/merged/release/values/values.xml:236: error: resource android:attr/fontVariationSettings not found.
  /Users/vagrant/git/build/flutter_stetho/intermediates/res/merged/release/values/values.xml:237: error: resource android:attr/ttcIndex not found.
  error: failed linking references.
```

Any ideas?

**Comment by andrewpmoore** [comment]
Change the android/build.gradle compileSdkVersion from 27 to 28 seems to fix this. I just came on here to raise the bug with this suggestion to fix it when I spotted yours.

**Comment by dustin-graham** [comment]
@Sleevezipper87 and @andrewpmoore I forked and fixed the issue in my project, made a PR^.

**Comment by stepankuzmin** [comment]
Same issue here. I've tried to change `compileSdkVersion` to `28`/`29`, but it didn't help.

**Comment by andrewpmoore** [comment]
@stepankuzmin did you change your compile version or the actual library compile version. You need to modify the actual library until there's an update for this for it to work

**Comment by maxgmer** [comment]
when will there be an update for the lib? same issue

**Comment by brianegan** [comment]
Hey all -- no guarantees on time, please have patience -- I'm on vacation visiting family for the holidays. I'll get to it when I'm back in action!

**Comment by brianegan** [comment]
published 0.5.2 with the fix. Thanks all!
